tasks/main.yml

```diff
@@ -22,3 +22,20 @@
 - name: Create /etc/passwd.client
   template: src=passwd.client dest=/etc/exim4/passwd.client group=Debian-exim
   notify: update exim4.conf
+
+- name: Add headers to outgoing emails
```

Comment:
This task name is misleading. The created configuration files are not only used for adding headers (per their own documentation).